calabash-cucumber/.irbrc

```diff
@@ -29,8 +29,8 @@ IRB.conf[:PROMPT_MODE] = :CALABASH_IOS
 
 begin
   require 'pry'
-  Pry.config.history.should_save = false
-  Pry.config.history.should_load = false
+  Pry.config.history_save = true
+  Pry.config.history_load = true
```

Comment:
Hey @JoeSSS!
Thanks a lot for your contribution, I really appreciate it.
These changes looks good to me but I have one question before I'll press `Approve`: Could you please explain why have you changed values of config from `true` to `false`?

Reply:
oh.. copy-paste error.. thanks for noticing